Backpacks.cs

```diff
@@ -581,6 +613,63 @@ private bool VerifyCanOpenBackpack(BasePlayer looter, ulong ownerId)
             return true;
         }
 
+        void OnPlayerConnected(BasePlayer player)
+        {
+            CreateGUI(player);
+        }
+
+        void OnPlayerSleepEnded(BasePlayer player)
+        {
+            CreateGUI(player);
+        }
+
+        void CreateGUI(BasePlayer player)
+        {
+            if (player == null || player.IsNpc)
+                return;
+
+            if (!permission.UserHasPermission(player.UserIDString, GUIPermission))
+                return;
+
+            CuiHelper.DestroyUi(player, GUIPanelName);
+            var elements = new CuiElementContainer();
+            var BackpacksUIPanel = elements.Add(new CuiPanel
+            {
+                Image = { Color = _instance._config.GUI.Color },
+                RectTransform = {
+                        AnchorMin = _config.GUI.GUIButtonPosition.Anchors.AnchorsMin,
+                        AnchorMax = _config.GUI.GUIButtonPosition.Anchors.AnchorsMax,
+                        OffsetMin = _config.GUI.GUIButtonPosition.Offsets.OffsetsMin,
+                        OffsetMax = _config.GUI.GUIButtonPosition.Offsets.OffsetsMax
+                    },
+                CursorEnabled = false
+            }, "Overlay", GUIPanelName);
+
+            elements.Add(new CuiElement
+            {
+                Parent = GUIPanelName,
+                Components = {
+                    new CuiRawImageComponent { Url = _instance._config.GUI.Image },
+                    new CuiRectTransformComponent { AnchorMin = "0 0", AnchorMax = "1 1" }
+                    }
+            });
+
+            elements.Add(new CuiButton
+            {
+                Button = {
+                    Command = "backpack.open", Color = "0 0 0 0" },
+                    RectTransform = { AnchorMin = "0 0", AnchorMax = "1 1" },
+                    Text = { Text = "" }
```

Comment:
nitpick: Move the `Command` part to the same line as `Button`, and unindent the other two lines one tab.